dask/array/ufunc.py

```diff
@@ -122,6 +122,7 @@ def outer(self, A, B, **kwargs):
 floor_divide = ufunc(np.floor_divide)
 negative = ufunc(np.negative)
 power = ufunc(np.power)
+float_power = ufunc(np.float_power)
```

Comment:
Apparently this was added in NumPy 1.12. So is not present in versions prior. Hence this will fail on older NumPy's. Trying to fix that in PR ( https://github.com/dask/dask/pull/2969 ).

Reply:
Should add CI didn't catch it in the PR as the oldest NumPy we test in the PR is 1.12.